_rest_config.php

```diff
@@ -216,20 +216,23 @@ public static function verifyAccessToken()
 
     public static function isTrustedUser($clientId, $userId)
     {
+        $trustedUserService = new \OpenEMR\Services\TrustedUserService();
```

Comment:
would place this in the `use` block at top and then use short form

Reply:
ok, this will go up in my next push.